lib/sqlalchemy/sql/selectable.py

```diff
@@ -6687,7 +6728,7 @@ def correlate(
         """
         e = self._clone()
         e.element = self._regroup(
```

Comment:
**mike bayer** (zzzeek) wrote:

remoive

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325